cmd/clusterctl/client/cluster/mover.go

```diff
@@ -436,6 +459,17 @@ func (o *objectMover) fromDirectory(graph *objectGraph, toProxy Proxy) error {
 	// - then all the MachineSets, then all the Machines, etc.
 	moveSequence := getMoveSequence(graph)
 
+	// Check if there are any nodes that were skipped from the move sequence and log them.
+	// This happens if the node has owners that will not be moved.
+	if len(moveSequence.skippedNodes) > 0 {
+		nodeNames := []string{}
+		for _, n := range moveSequence.skippedNodes {
+			nodeNames = append(nodeNames, fmt.Sprintf(" * %s", n.identityStr()))
+		}
+		warningMsg := fmt.Sprintf("[Warning] The following objects will not be moved because they are owned by objects that are not part of the move sequence: \n%s", strings.Join(nodeNames, "\n"))
+		log.V(1).Info(warningMsg)
+	}
```

Comment:
I would say this cannot happen because fromDirectory reads the input of toDirectory